pandas/io/common.py

```diff
@@ -458,12 +458,12 @@ def _get_handle(
                     f = zf.open(zip_names.pop())
                 elif len(zip_names) == 0:
                     raise ValueError(
-                        "Zero files found in ZIP file {}".format(path_or_buf)
+                        f"Zero files found in ZIP file {path_or_buf}"
                     )
                 else:
                     raise ValueError(
-                        "Multiple files found in ZIP file."
-                        " Only one file per ZIP: {}".format(zip_names)
+                        f"Multiple files found in ZIP file."
+                        " Only one file per ZIP: {zip_names}"
```

Comment:
```suggestion
                        f" Only one file per ZIP: {zip_names}"
```
`f` needed on this line